internal/cmd/schema-registry/command_config_describe.go

```diff
@@ -15,7 +17,12 @@ import (
 )
 
 type configOut struct {
-	CompatibilityLevel string `human:"Compatibility Level" serialized:"compatibility_level"`
+	CompatibilityLevel string `human:"Compatibility Level,omitempty" serialized:"compatibility_level,omitempty"`
+	CompatibilityGroup string `human:"Compatibility Group,omitempty" serialized:"compatibility_group,omitempty"`
+	DefaultMetadata    string `human:"Metadata Defaults,omitempty" serialized:"metadata_defaults,omitempty"`
+	OverrideMetadata   string `human:"Metadata Overrides,omitempty" serialized:"metadata_overrides,omitempty"`
+	DefaultRuleSet     string `human:"Ruleset Defaults,omitempty" serialized:"ruleset_defaults,omitempty"`
```

Comment:
Nit: "ruleset" is one word.
```suggestion
	RulesetDefaults     string `human:"Ruleset Defaults,omitempty" serialized:"ruleset_defaults,omitempty"`
```